### ISSUE TITLE
Blurred image not positioned correctly when using object-position

### ISSUE THREAD

**Issue by schoenwaldnils** [vercel/next.js#26309]
### What version of Next.js are you using?

11.0.0

### What version of Node.js are you using?

14.13.1

### What browser are you using?

Chrome

### What operating system are you using?

macOS

### How are you deploying your application?

Vercel

### Describe the Bug

Blurred images are not respecting the objectPosition property in Image component

Full image: [Example Image](https://i.schoen.world/cVAqU.png)

**What I see:** 
![](https://i.schoen.world/g6CeV.png)

### Expected Behavior

The blurred placeholder should have the save position as the the later loaded full image.

**What I expect:**

![](https://i.schoen.world/6ctkR.png)

An easy fix for this would be to mirror the `object-position` value to the `background-position` value while loading.

### To Reproduce

Setup:

```jsx
import Image from 'next/image'

import exampleImg from '../data/example.png'

const Page = () => (
  <Image
    src={exampleImg}
    width={1024}
    height={700}
    layout="responsive"
    objectFit="cover"
    objectPosition="center"
    placeholder="blur"
  />
)

export default Page
```

Deployed example: [schoenwaldnils.vercel.app/objectPositionIssue](https://schoen-world-8793xnvzh-schoenwaldnils.vercel.app/objectPositionIssue)

**Comment by styfle** [comment]
Hi @schoenwaldnils 

Thanks for reporting this issue and providing great repro steps!

It sounds like you already came up with a solution, would you like to submit a PR to fix it? 🙂

**Comment by DarkCode01** [comment]
i would like to make this issues 😀 @styfle @schoenwaldnils

**Comment by schoenwaldnils** [comment]
@DarkCode01 you mean the PR? 
Go ahead, I didn't started yet 😁

**Comment by schoenwaldnils** [comment]
@DarkCode01 how is your PR going? https://github.com/vercel/next.js/pull/26327

**Comment by DarkCode01** [comment]
> @DarkCode01 how is your PR going? #26327

when `Image` recive the prop `objectPosition`, `backgroundPosition` is added with the same value as `objectPosition`.

**Comment by schoenwaldnils** [comment]
@DarkCode01 I rather means how the progress is ^^ I saw that there is a comment on the PR that there are two file changes that should not be needed

**Comment by styfle** [comment]
The fix is available in [next@11.0.2-canary.4](https://github.com/vercel/next.js/releases/tag/v11.0.2-canary.4), thanks!

**Comment by balazsorban44** [comment]
This issue has been automatically locked due to no recent activity. If you are running into a similar issue, please create a new issue with the steps to reproduce. Thank you.
